src/System.Runtime/tests/System/GCTests.cs

```diff
@@ -112,12 +119,17 @@ public static void KeepAlive()
 
         private class KeepAliveTest
         {
+            [System.Runtime.CompilerServices.MethodImplAttribute(System.Runtime.CompilerServices.MethodImplOptions.NoInlining)]
+            private static void MakeAndDropDNKA()
+            {
+                new DoNotKeepAliveObject();
```

Comment:
No real reason, really - it's kind of weird now that GC refs needs to be local to a method to be collectible for sure. If you guys don't mind please just let me know what you want me to do and I promise to do it ;). Thanks.